test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-large-cluster.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 720
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=180m -port jmx=6868 -mode cql3 native -rate threads=50",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=180m -port jmx=6868 -mode cql3 native -rate threads=50",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=180m -port jmx=6868 -mode cql3 native -rate threads=50",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=180m -port jmx=6868 -mode cql3 native -rate threads=50"
+            ]
+
+n_db_nodes: '15 15 15'
+instance_type_db: 'i3.large'
+n_loaders: '2 2 2'
+region_aware_loader: true
+n_monitor_nodes: 1
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_include_filter: ['topology_changes']
+nemesis_seed: '023'
+nemesis_interval: 5
+nemesis_add_node_cnt: 3
+
+round_robin: true
+
+user_prefix: 'longevity-cdc-100gb-3h-multidc-large-cluster'
```

Comment:
this doesn't fit the file name (or the file name doesn't fit this parameter

Reply:
On one hand the stress is 3h, OTOH the job is set to 8h.
Not sure what's better, for now it's good enough.